jacoco_shaded/pom.xml

```diff
@@ -32,4 +32,16 @@
     <url>https://github.com/thought-machine/please-java</url>
   </scm>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>11</source>
+          <target>11</target>
```

Comment:
The target language level should be lower than this - we should be able to target 7 or 8.